.github/workflows/label-pr.yml

```diff
@@ -19,8 +19,11 @@ on:
       - auto_merge_enabled
       - auto_merge_disabled
       - ready_for_review
-  pull_request_review:
-    types: [submitted]
+  workflow_run:
+    workflows:
+      - PR Review
+    types:
+      - completed
```

Comment:
It might be worth it to mention in a comment here that this is a workaround and to reference the `label-pr-review.yml` file for more details?